src/server/pps/cmds/cmds.go

```diff
@@ -1,15 +1,15 @@
 package cmds
 
 import (
-	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 	"text/tabwriter"
 
+	"github.com/Jeffail/gabs"
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/pachyderm/pachyderm"
-	"github.com/pachyderm/pachyderm/src/client"
+	pach "github.com/pachyderm/pachyderm/src/client"
```

Comment:
I've been using `c` for the variable name to avoid this conflict. But this works too.